packages/iframe-stamper/src/index.ts

```diff
@@ -230,6 +239,65 @@ export class IframeStamper {
     });
   }
 
+  async injectImportBundle(bundle: string): Promise<boolean> {
+    this.iframe.contentWindow?.postMessage(
+      {
+        type: IframeEventType.InjectImportBundle,
+        value: bundle,
+      },
+      "*"
+    );
+
+    return new Promise((resolve, reject) => {
+      window.addEventListener(
+        "message",
+        (event) => {
+          if (event.origin !== this.iframeOrigin) {
+            // There might be other things going on in the window, for example: react dev tools, other extensions, etc.
+            // Instead of erroring out we simply return. Not our event!
+            return;
+          }
+          if (event.data?.type === IframeEventType.BundleInjected) {
+            resolve(event.data["value"]);
+          }
+          if (event.data?.type === IframeEventType.Error) {
+            reject(event.data["value"]);
+          }
+        },
+        false
+      );
+    });
+  }
+
+  async extractWalletEncryptedBundle(): Promise<string> {
```

Comment:
I think this is tightly coupled to this discussion: https://github.com/tkhq/frames/pull/25#discussion_r1506786320. But I'm happy to get this merge as-is and change this as a followup (if we decide to change it, that is)